File: Proyects/UniJava/TP4/ex10/Call.java
public class Call {

   protected String from, to;
   private int duration;

   protected double cost = 98;

   Call(String from, String to, int duration) {
       this.from = from;
       this.to = to;
       this.duration = duration;
   }

   public double getCost() {
       return duration * cost;
   }

   public void setCost(int cost){
	this.cost = cost;
   }

}

File: Proyects/UniJava/TP10/ex4/call.rb
class Call
  COST_PER_SECOND = 1

  def initialize(from, to, duration)
    @from = from
    @to = to
    @duration = duration
  end

  def to
    @to
  end

  def cost
    @duration * COST_PER_SECOND
  end
end
